moonbeam-types-bundle/index.ts

```diff
@@ -336,6 +337,45 @@ const TYPES_800_undefined: RegistryTypes = {
   MultiLocation: "MultiLocationV1",
 };
 
+const TYPES_900_undefined_deprecated: RegistryTypes = {
+  ...TYPES_800_899,
```

Comment:
So this should be removed

Reply:
Why ?

Reply:
Jaco said we should inject empty object for post 900

Reply:
It won’t have adverse affects if it is populated, will be injected, but decoding won’t use it. (I had the same on Kusama/Polkadot/Westend/Rococo where I neglected to clear it)

Reply:
Ok I understand why this should be there as per other conv